src/compiler/checker.ts

```diff
@@ -1776,6 +1777,20 @@ namespace ts {
             }
         }
 
+        function tryGetMemberInModuleExportsAndProperties(memberName: __String, moduleSymbol: Symbol): Symbol | undefined {
+            const symbolTable = getExportsOfModule(moduleSymbol);
+            if (symbolTable) {
+                let symbol = symbolTable.get(memberName);
+                if (!symbol) {
+                    const exportEquals = resolveExternalModuleSymbol(moduleSymbol);

```

Comment:
At this point, I'll keep the functions separate in the interest of keeping the public API stable. I am changing the code to have this one call ```tryGetMemberInModuleExports```